openapi-bal-service/src/main/java/io/ballerina/openapi/converter/service/OpenAPIComponentMapper.java

```diff
@@ -18,24 +18,7 @@
 
 package io.ballerina.openapi.converter.service;
 
-import io.ballerina.compiler.api.symbols.ArrayTypeSymbol;
```

Comment:
Importing all the classes from a java package is an anti-pattern. So, only import relevant classes which are used in the code.